src/pybind/mgr/rook/module.py

```diff
@@ -12,6 +13,13 @@
     from kubernetes.client.rest import ApiException
 
     kubernetes_imported = True
+
+    # https://github.com/kubernetes-client/python/issues/895
+    from kubernetes.client.models.v1_container_image import V1ContainerImage
+    def names(self, names):
+        self._names = names
+    V1ContainerImage.names = V1ContainerImage.names.setter(names)
```

Comment:
This makes me slightly nervous, just because the default setter explicitly prevents names from being None, and I assume that was done for reason :-) Is it possible that something in the test environment doesn't have a name, but should? (Disclaimer: I don't know enough about the k8s side of things to be sure I'm asking the right questions)

Reply:
This occurs when deserializing JSON to Python objects when somehow, a `V1ContainerImage` is deserialized when reading the K8s nodes. Listing the nodes should not contain container images, thus we're not fixing the real bug here.

Reply:
Gotcha.

Reply:
See https://github.com/kubernetes-client/python/issues/895